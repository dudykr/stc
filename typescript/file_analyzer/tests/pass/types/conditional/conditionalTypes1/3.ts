export function f1<T>(x: T, y: NonNullable<T>) {
    x = y;
}