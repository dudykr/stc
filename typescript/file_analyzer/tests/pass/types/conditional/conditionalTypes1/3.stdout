warning: Type
 --> $DIR/tests/pass/types/conditional/conditionalTypes1/3.ts:2:9
  |
2 |     x = y;
  |         ^
  |
  = note: NonNullable<T>

warning: Type
 --> $DIR/tests/pass/types/conditional/conditionalTypes1/3.ts:2:5
  |
2 |     x = y;
  |     ^^^^^
  |
  = note: NonNullable<T>

